spec/definitions.yaml

```diff
@@ -1000,12 +1037,21 @@ definitions:
         description: Information on the current size and status of the node pool
         type: object
         properties:
+          instance_types:
```

Comment:
@marians I couldn't think of a good structure here which doesn't totally change this stuff. Do you have an idea?

Reply:
This mention went past me, sorry. Don't see the problem here actually.